Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN mkdir -p /code
 WORKDIR /code
 
 # Install global and local dependencies first so they can be cached.
-RUN npm install -g yarn@^1.17
+RUN npm install -gf yarn@^1.21.1
```

Comment:
While technically correct, in this kind of automation flow, I generally prefer the explicit forms of `--global` `--force`.

There's no reason to not be verbose, especially when a human has to review it if it explodes.